restful-tango/tangoREST.py

```diff
@@ -301,12 +302,11 @@ def upload(self, key, courselab, file, body):
             labPath = self.getDirPath(key, courselab)
             try:
                 if os.path.exists(labPath):
+                    fileMD5 = hashlib.md5(body).hexdigest()
+                    filesInDir = self.computeMD5(labPath)
+                    if file in filesInDir and filesInDir[file] == fileMD5:
+                        return self.status.file_exists
```

Comment:
So even if one file/directory is the same we return self.status.file_exists and stop the whole upload process?